src/server/v2.0/handler/repository.go

```diff
@@ -233,6 +234,7 @@ func (r *repositoryAPI) UpdateRepository(ctx context.Context, params operation.U
 	}
 	if err := r.repoCtl.Update(ctx, &repomodel.RepoRecord{
 		RepositoryID: repository.RepositoryID,
+		Name:         repository.Name,
```

Comment:
why it needs to specify the name?

Reply:
the manager needs to clean up cache when Update(), and it maybe has cache key by id and name, so if only pass the id, the name cache key will not be cleaned, although we can re-get repo inside Update but here above has called once so just pass it.